OpenSim/Common/Component.cpp

```diff
@@ -134,11 +152,19 @@ void Component::finalizeFromProperties()
 }
 
 // Base class implementation of virtual method.
-// Call extendFinalizeFromProperties on all components
+// Call finalizeFromProperties on all subcomponents
 void Component::componentsFinalizeFromProperties() const
```

Comment:
The names that are auto-generated are not unique increasing the potential of downstream problems with bogus messages (e.g. Path_ component is not found or is not unique etc.) while the user has no such name in his/her model. Is that possible?